examples/frame_namespaces.c

```diff
@@ -73,12 +73,12 @@ int main() {
     blosc2_frame frame1 = BLOSC_EMPTY_FRAME;
 
     // Add some namespaces
-    blosc2_frame_add_namespace(&frame1, "my_namespace1", (uint8_t *) "my_content1",
-                               (uint32_t) strlen("my_content1"));
-    blosc2_frame_add_namespace(&frame1, "my_namespace2", (uint8_t *) "my_content1",
-                               (uint32_t) strlen("my_content1"));
-    blosc2_frame_update_namespace(&frame1, "my_namespace2", (uint8_t *) "my_content2",
-                               (uint32_t) strlen("my_content2"));
+  blosc2_frame_add_metalayer(&frame1, "my_namespace1", (uint8_t *) "my_content1",
```

Comment:
For consistency, one should rename `my_namespace*` -> `my_metalayer*` all througout the file.